src/backend/vulkan/src/device.rs

```diff
@@ -1891,11 +1891,25 @@ impl d::Device<B> for super::Device {
     }
 
     fn start_capture(&self) {
-        //TODO: RenderDoc
+        unsafe {
+            match &self.shared.instance.render_doc_entry {
```

Comment:
Instead of matching on a reference, let's match on the actual thing. We'd then have `Ok(ref entry)` and `Err(ref cause)` cases